### PR TITLE
add ophan tracking

### PR DESCRIPTION
**Pull request by gidsg**
CC @philwills @gtrufitt 
screen shot 2016-06-14 at 13 51 33

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,24 +14,25 @@ <h2 class="fieldset__headline title"></h2>
...
             <div class="actions">
-                <button class="submit button button--primary button--large">Vote</button>
+                <button class="submit button button--primary button--large" data-link-name="poll-submit">Vote</button>
             </div>
         </div>
     </fieldset>
 </form>
 </body>
 <script src="bundle.js"></script>
+<script src="//j.ophan.co.uk/interactive.js"></script>
```
Comment (gtrufitt) [inline index.html]: Sweet! That is useful.

File: index.html [hunk 2]
```diff
@@ -14,24 +14,25 @@ <h2 class="fieldset__headline title"></h2>
...
-                    <input type="radio" class="pseudo-radio-input" name="option" id="q2-input" value="option-two" required="required">
+                    <input type="radio" class="pseudo-radio-input" name="option" id="q2-input" value="option-two" required="required" data-link-name="poll-option">
                     <div class="pseudo-radio">
                         <div class="q2"></div>
                     </div>
                 </label>
             </div>
             <div class="actions">
-                <button class="submit button button--primary button--large">Vote</button>
+                <button class="submit button button--primary button--large" data-link-name="poll-submit">Vote</button>
```
Comment (gtrufitt) [inline index.html]: I think it's worth passing a bit more data here - maybe we could parse in the poll id - link names are usually separated by `:` so you could do `pollId : poll : poll-submit` - that way we can filter either on poll id, polls in general or the specific event. Also, you could do  `pollId : poll : poll-option : poll-option-1` etc which would give us the option to look further into the submissions (though not much more useful than just making sure everyone isn't submitting just on the first value?)

**Comment by philwills** [comment]
Looks good from the Ophan implementation side.

File: index.html (hunk 2 above)
Comment (gidsg) [inline index.html]: I asked @philwills about this and I think the feeling is that the poll ID would be redundant as we can derive this by the URL. Agree the `data-link-name` `poll-option` option doesn't add much as-is.

Comment (gtrufitt) [inline index.html]: Sure thing. 👍

Comment (gidsg) [inline index.html]: OK @gtrufitt I have marked up the options now as you suggested e.g. `"poll-id : hilarypoll : poll-option : no"`

**Pull request merged**